### ISSUE TITLE
Issue with lib: Russia coords not resolved

### ISSUE THREAD

**Issue by Radzhab** [InquisitorJax/Wibci.CountryReverseGeocode#2]
I try get info by this coordinates but its not find it
Lon
40.587502313216
lat
50.064546356068
```
GeoLocation country = new GeoLocation {Longitude = lon, Latitude = lat};
CountryReverseGeocodeService _service = new CountryReverseGeocodeService();
var info = _service.FindCountry(country);

**Comment by InquisitorJax** [comment]
it seems there's an issue with that coordinate - Russia right?
I tried adding more accurate data from https://gist.github.com/neilkennedy/6423661 and https://raw.githubusercontent.com/datasets/geo-boundaries-world-110m/master/countries.geojson
and replacing the polygon data in the CountryData class
... but it still didn't find it.

**Comment by Radzhab** [comment]
Yeap, Russia.  Why it?

**Comment by Radzhab** [comment]
How solve it?

**Comment by InquisitorJax** [comment]
I'm not sure. Unfortunately I don't have the time to have a look in the short term. Feel free to fork the source code and have a look ;)

**Comment by InquisitorJax** [comment]
fixed in 1.2 release. added specific test for russia coords.
1.1 still had old 1.0 assembly reference :(
